tests/integration/test_worker.py

```diff
@@ -22,7 +23,8 @@ def test_duplicate_id(self):
 
     def test_reregister_id(self):
         worker_id = self.register_worker(self.get_header(), expected_code=200)
-        time.sleep(HEARTBEAT_INTERVAL * 2 + 1)
+        time.sleep(self.app.settings["FLAGS"]["heartbeat_interval"] * 2 + 1)
```

Comment:
This test will take a while to run (21 seconds). Can you set the heartbeat to be 1 second in `AsyncHTTPMixin:get_app()`.

Reply:
weird.. it only took 3 seconds locally.

Reply:
never mind, the heartbeat is set to 1 (in tests/_fixtures/config.json which is used to initialize the test). all good.